doc/whats_new/v0.20.rst

```diff
@@ -382,6 +382,11 @@ Linear, kernelized and related models
   underlying implementation is broken. Use :class:`linear_model.Lasso` instead.
   :issue:`9837` by `Alexandre Gramfort`_.
 
+- :class:`linear_model.LogisticRegression` with ``solver='lbfgs'`` formerly
```

Comment:
I doubt whether it belongs to `API changes`, either `Enhancements` or `Bug fixes` seems fine from my side.

Reply:
Going from scikit-learn 0.19 to 0.20, `logistic_regression.n_iter_` will change (unless you use scipy > 1.0.0). In this respect "API changes" feels like the right place.